packages/playwright-test/src/common/transform.ts

```diff
@@ -155,6 +164,18 @@ export function transformHook(code: string, filename: string, moduleUrl?: string
   }
 }
 
+function calculateHash(content: string, filePath: string, isModule: boolean, pluginsPrologue: BabelPlugin[], pluginsEpilogue: BabelPlugin[]): string {
```

Comment:
Have we seen compiler producing different outcome depending on the version of Node? That would mean there is a bug in Node since the compilation is supposed to be deterministic.